stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/avatar/ImageAvatar.kt

```diff
@@ -0,0 +1,50 @@
+package io.getstream.chat.android.compose.ui.components.avatar
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.material.ripple.rememberRipple
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.layout.ContentScale
+import io.getstream.chat.android.compose.ui.theme.ChatTheme
+
+/**
+ * Avatar/image that renders whatever image the [painter] provides. It allows for customization,
```

Comment:
I would rephrase the first sentence to:

`Renders an image the [painter] provides.`